b288702/README.md

```diff
@@ -28,23 +28,36 @@ The following are requirements for participating in Filecoin's mainnet Phase 2.
 
 1. Build `rust-fil-proofs`, please also check its [Rust installation/build instructions](https://github.com/filecoin-project/rust-fil-proofs/blob/6e38487293a2ec063688acb4a414600b1c0654f9/README.md#install-and-configure-rust):
 
+You will check out the commit corresponding to the [filecoin-proofs-v5.1.1](https://github.com/filecoin-project/rust-fil-proofs/releases/tag/filecoin-proofs-v5.1.1) tag, named explicitly below to ensure you are using the authoritative verification code.
 ```bash
 $ git clone https://github.com/filecoin-project/rust-fil-proofs.git
 $ cd rust-fil-proofs
-$ git checkout 6e38487
+$ git checkout a700f68
 $ cargo build --release --bin phase2 && cp target/release/phase2 .
 ```
 
 2. Download checksums file and verification script:
 
 ```bash
 $ curl -O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/b80d1f8/b288702/b288702.b2sums \
--O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/b80d1f8/b288702/verify_all.sh && chmod +x verify_all.sh
+-O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/b80d1f8/b288702/verify_all.sh \
+-O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/b80d1f8/b288702/verify_initial.sh \
+-O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/b80d1f8/b288702/verify_contrib.sh \
+-O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/b80d1f8/b288702/verify_final.sh \
```

Comment:
I think that "b80d1f8" needs to be removed from these urls.

Reply:
We wanted to point to a specific commit and not to master. hence we used `b80d1f8`. On GitHub you can also use short hashes.

Reply:
Oh, it's a permalink not a directory, that makes sense. Usually permalinks use the full 20 byte sha1 digest, so I was confused